RM_Vk/RM_Vk/Model/Group.swift

```diff
@@ -0,0 +1,51 @@
+// Group.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Генерация групп
```

Comment:
```suggestion
/// Генерация групп 
```
Дока не соответсвует названию